Makefile

```diff
@@ -1045,7 +1045,7 @@ toplevel/native/topeval.cmx: otherlibs/dynlink/dynlink.cmxa
 make_opcodes := tools/make_opcodes$(EXE)
 
 bytecomp/opcodes.ml: runtime/caml/instruct.h $(make_opcodes)
-	runtime/ocamlrun$(EXE) $(make_opcodes) -opcodes < $< > $@
+	$(NEW_OCAMLRUN) $(make_opcodes) -opcodes < $< > $@
```

Comment:
cf. https://github.com/ocaml/ocaml/pull/10293. This alteration doesn't affect that problem (either good or bad). Note though that I have as part of fixing something else implemented the suggestion in https://github.com/ocaml/ocaml/pull/10293#issuecomment-797732980 of moving `tools/make_opcodes` into `ocamlc` (so `boot/ocamlc` is capable of doing this) which might simplify things further?